search/django/weedid/views.py

```diff
@@ -76,10 +77,10 @@ def upload(request):
         images = []
         file_weedcoco = request.FILES["weedcoco"]
         weedcoco_json = json.load(file_weedcoco)
-        validate(
-            weedcoco_json,
-            schema=request.POST["schema"] if request.POST["schema"] else "coco",
-        )
+        # validate(
```

Comment:
Is this due to the problem on your local dev machine with the EPPO data having an invalid Zip?

Reply:
Yeah I'll revert it back